src/cartservice/Program.cs

```diff
@@ -12,116 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using CommandLine;
-using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Hosting;
-using System.Runtime.CompilerServices;
+using cartservice;
 
-[assembly: InternalsVisibleTo("cartservice.tests")]
-namespace cartservice
-{
-    public class Program
-    {
-        const string REDIS_ADDRESS = "REDIS_ADDR";
+CreateHostBuilder(args).Build().Run();

```

Comment:
I'm not sure I understand how this even compiles - it looks like you are removing the namespace and class and this method is not contained anywhere. Is it a glitch in the comparison?

Reply:
Good question, that's one of the improvements with C# 9 described here: https://devblogs.microsoft.com/dotnet/welcome-to-c-9-0/#top-level-programs.

Reply:
Let's revert to have a main method. .NET documentation/tutorials throughout the internet haven't picked up all the new features from the .NET team and many of the customers might find this confusing.

Reply:
Done.